src/components/shared/chart/Viewport.js

```diff
@@ -463,6 +513,94 @@ export const withChartViewport: WithChartViewport<*, *> =
         this.moveViewport(offsetX, offsetY);
       };
 
+      _keyDownListener = (
+        event: { nativeEvent: KeyboardEvent } & SyntheticKeyboardEvent<>
+      ) => {
+        let navigationKey;
+        if (
+          !event.ctrlKey &&
+          !event.shiftKey &&
+          !event.altKey &&
+          !event.metaKey
+        ) {
+          navigationKey = BARE_KEYMAP[event.nativeEvent.code];
+        } else if (event.ctrlKey) {
+          navigationKey = CTRL_KEYMAP[event.nativeEvent.code];
```

Comment:
Could you elaborate with some code comments what's going on with the CTRL_KEYMAP here? Also, could you explain why CTRL is in the mix here? At least on my Mac I can't get ctrl to do anything other than activate keyboard shortcuts.

Reply:
Sure, I'll add some comments. Ctrl + arrow up/down should zoom. Does it work for you, or is it different on Mac?